### PR TITLE
Added electron ID

### PR DESCRIPTION
**Pull request by bainbrid**

### CODE REVIEW

File: BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc [hunk 1]
```diff
@@ -46,19 +51,25 @@ void PATLowPtElectronSeedingEmbedder::produce(edm::StreamID, edm::Event &evt, ed
...
   evt.getByToken(unBiased_src_, unBiased);
+  edm::Handle<edm::ValueMap<float> > mvaId;  
+  evt.getByToken(mvaId_src_, mvaId);
 
 
   std::unique_ptr<pat::ElectronCollection> out(new pat::ElectronCollection);
 
-  for(auto ele : *lowpt) {
+  for ( size_t iele = 0; iele < lowpt->size(); ++iele ) {
+    edm::Ptr<pat::Electron> ptr(lowpt,iele);
```
Comment (mverzett) [inline BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc]: Why Ptr and not Ref?

Comment (bainbrid) [reply BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc]: There's no particular reason here, but the choice was biased by my ntuplizer code, which requires a edm::Ptr to correctly deal with both reco::GsfElectrons and pat::Electrons, via an edm::View in the handle (to either collection) ...
I can change to a Ref if you prefer? Do we gain anything?

Comment (mverzett) [reply BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc]: I don't think so, just out of curiosity. We can keep it like this

File: BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc [hunk 2]
```diff
@@ -58,15 +58,15 @@ void PATLowPtElectronSeedingEmbedder::produce(edm::StreamID, edm::Event &evt, ed
   std::unique_ptr<pat::ElectronCollection> out(new pat::ElectronCollection);
 
   for ( size_t iele = 0; iele < lowpt->size(); ++iele ) {
-    edm::Ptr<pat::Electron> ptr(lowpt,iele);
-    pat::Electron ele = *ptr;
-    const reco::GsfTrackRef gsfTrk = ele.gsfTrack();
+    edm::Ref<pat::ElectronCollection> ref(lowpt,iele);
```
Comment (bainbrid) [inline BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc]: @mverzett switched to Ref, as it's more familiar to most.

File: BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc [hunk 3]
```diff
@@ -46,19 +51,25 @@ void PATLowPtElectronSeedingEmbedder::produce(edm::StreamID, edm::Event &evt, ed
...
   std::unique_ptr<pat::ElectronCollection> out(new pat::ElectronCollection);
 
-  for(auto ele : *lowpt) {
-    const reco::GsfTrackRef gsfTrk = ele.gsfTrack();
+  for ( size_t iele = 0; iele < lowpt->size(); ++iele ) {
+    edm::Ref<pat::ElectronCollection> ref(lowpt,iele);
+    const reco::GsfTrackRef gsfTrk = ref->gsfTrack();
     float unbiased_seedBDT = float((*unBiased)[gsfTrk]);
     float ptbiased_seedBDT = float((*ptBiased)[gsfTrk]);
+    float mva_id = float(mvaId->get(ref.key()));
```
Comment (mverzett) [inline BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc]: @bainbrid then maybe we can go for a simpler ```(*mvaId)[ref]```

Comment (bainbrid) [reply BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc]: Done. PR is updated.

**Pull request merged**